azure.yaml

```diff
@@ -8,16 +8,30 @@ services:
     project: ./app/backend
     language: py
     host: appservice
+  scripts:
+    project: ./scripts
+    language: py
+    host: ""
 hooks:
     postprovision:
-      shell: pwsh
-      run: ./scripts/prepdocs.ps1
-      interactive: true
-      continueOnError: false
+      windows:
+        shell: pwsh
+        run: ./scripts/prepdocs.ps1
+        interactive: true
+        continueOnError: false
+      posix:
+        shell: sh
+        run: scripts/scripts_env/bin/python scripts/prepdocs.py 'data/*' --storageaccount $AZURE_STORAGE_ACCOUNT --container $AZURE_STORAGE_CONTAINER --searchservice $AZURE_SEARCH_SERVICE --index $AZURE_SEARCH_INDEX -v
```

Comment:
For consistency, I'd like to create a .sh file for this.

Reply:
Makes sense.